slack-clone-compose-sample/src/main/kotlin/io/getstream/slack/compose/ui/theme/Color.kt

```diff
@@ -0,0 +1,8 @@
+package io.getstream.slack.compose.ui.theme
+
+import androidx.compose.ui.graphics.Color
+
+val Purple200 = Color(0xFFBB86FC)
```

Comment:
I think we can remove these colors that are generated once you get working on the app!